src/tests/ftest/container/attribute.yaml

```diff
@@ -4,6 +4,8 @@ hosts:
   test_servers:
     - server-A
 timeout: 50
+server_manager:
+  recreate: true
```

Comment:
Does this add --recreate-superblocks to command line? Aren't we supposed to remove them and start performing --reformat for both tmpfs and dcpm?

Reply:
Yes I believe it does. It is a consequence of this very recent PR that now requires all tests that use recreate-superblocks on server start to specifically opt-in
https://github.com/daos-stack/daos/commit/81a26c4987732b6a19f19f26516a82b8fee74f31

It is not a goal of this patch to fix the infrastructure for launching tests. There is another PR in progress as I understand to address server launch issues.